ports/antonballclassic/port.json

```
@@ -0,0 +1,31 @@
+{
+  "version": 3,
+  "name": "antonballclassic",
```

It's the zip archive name with extension.
```suggestion
  "name": "antonballclassic.zip",
```